apps/rush-lib/src/cli/logic/InstallManager.ts

```diff
@@ -285,6 +287,10 @@ export default class InstallManager {
     fsx.removeSync(tempNpmrcPath);
     this.syncFile(committedNpmrcPath, tempNpmrcPath);
 
+    if (fsx.existsSync(tempNpmrcPath)) {
+      fsx.appendFileSync(tempNpmrcPath, [''].concat(authTokens).join(os.EOL));
```

Comment:
>[''].concat( [](start = 40, length = 12)

is this necessary? you set authTokens to [] already

Reply:
I want to make sure what we append starts with a newline so we don't accidentally write something to the end of an existing line.

---
In reply to: [177545268](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r177545268) [](ancestors = 177545268)